modules/UI/videolayout/VideoLayout.js

```diff
@@ -123,14 +123,32 @@ var VideoLayout = {
     },
 
     setAudioLevel(id, lvl) {
+        var interactive = interfaceConfig.INTERACTIVE_MICROPHONE;
+
         if (!largeVideo) {
             return;
         }
+
+        if (interactive && this.getVideoThumbnailById(id)) {
+            this.getVideoThumbnailById(id).imic.volume(lvl * 100).redraw();
+            return;
+        }
+
         AudioLevels.updateAudioLevel(
             id, lvl, largeVideo.id
         );
     },
 
+    getVideoThumbnailById(id) {
```

Comment:
JSDocs missing. Also if we introduce this function we should make sure to use wherever possible to get a video object.